homeassistant/config_entries.py

```diff
@@ -891,6 +891,7 @@ def _abort_if_unique_id_configured(
         self,
         updates: Optional[Dict[Any, Any]] = None,
         reload_on_update: bool = True,
+        allow_ignored_devices: bool = False,
```

Comment:
I wonder if this is really needed. We are aware of the running flow at this point, so we should be able to check the current origin/source of this running flow. I guess it is safe to say, that if the source is the user; we could default to this behavior.

The advantage would be, that this can be implemented for all; instead of requiring each existing config flow to adopt.

Reply:
I kept this flag for backwards compatibility. I think at this point it's hard to determine how this will affect other integrations. Although, I am excited by the option this fix can go global.

Reply:
I will ship this fix and get the comments from reviewers

Reply:
I think we should resolve this in this PR and not introduce options we'd have to remove again later.

If a config flow is originating from a user action, we can choose to skip existing entries by default.